### ISSUE TITLE
Strange Odd Stack Count (Ghosted) On Normal Use of the Game (Without touching Whereisit).

### ISSUE THREAD

**Issue by KaneHart** [JackFred2/ChestTracker#43]
So the only change I made was added the API and your Mod and since then at random sometimes multiple items in a chest sometimes none in fact most of the time none of them error out. But when they do it has this odd number count like in the image below:

![image](https://user-images.githubusercontent.com/425853/146830865-57ae8710-49da-4811-9cf6-9ab8d4c52e30.png)

The First Tin Cable. 1134 Item Count. When I click it then auto cleans it up. 

If you add up all the Tin Cable it comes to 1134. So I assume it might be something running when it should not run :)

For a list of other mods I use:
https://www.curseforge.com/minecraft/modpacks/modern/relations/dependencies

Thanks.

**Comment by JackFred2** [comment]
Just to be clear, you installed ChestTracker and Cloth API and this occured?

**Comment by KaneHart** [comment]
Correct. I can go ahead and remove it tomorrow from my own instance and see if it stops just in case. But those were the only 2 changes pure client side no server side and playing MP.
